### DISCUSSION TITLE
2022 Working Groups - Planning Thread

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#653] [General]
We're doing some 2022 planning and want to focus on a few working groups focused on chart development for different topics, including: 

- [QT Graphics](https://github.com/SafetyGraphics/safetyCharts/pull/90) - Mengchun / Xiao
- [Nep Explorer](https://github.com/SafetyGraphics/nep-explorer) - Jim / ? 
- [Volcano Plot](https://github.com/SafetyGraphics/volcanoPlot) - Dennis / ? 
- [safetyCharts v1.0](https://github.com/SafetyGraphics/safetyCharts) - ? / Jeremy 

Some logistics: 
- I've listed possible Clinical and Technical leads for each group
- We'd like to have at least 4 "core"  core contributors to each of these work streams.
- Each team will organize regular meetings and report back to our monthly full team meetings regularly. 

cc: @xni7 @mli1 @JimBuchanan

**Comment by JimBuchanan** [comment]
Thanks for pulling this together, Jeremy. I am also bringing Siu-chi Sun in to help me with the Nep Explorer.

**Comment by jwildfire** [comment]
I've added a question regarding sub-teams to our poll here: 

https://docs.google.com/forms/d/e/1FAIpQLSd4wkSfaajuj3iaVHirqj8L05S3yH7GVGwopufWRMmjif4IDg/viewform?usp=sf_link

We'd like everyone on the working group to fill out the form to specify your areas of interest for next year.
